src/locale/en.json

```diff
@@ -7,7 +7,8 @@
     "Others": "Others",
     "Swift Search": "Swift Search",
     "Swift Search API": "Swift Search API",
-    "Symphony Logo": "Symphony Logo"
+    "Symphony Logo": "Symphony Logo",
+    "Ok": "Ok"
```

Comment:
Crowdin is not configured on SDA, we also need to add the different entries in Fr and JP i18n files

Reply:
Possibly, but lets ask for the translation from Crowdin so I can manually add it. I have also added few more keys, where should I send these keys to get the translation?

Reply:
Awaiting for new translation @sbenmoussati . Shoud I ping in Crowdin room?